Worktez/worktez
# Fix (edit-page)/updated packages

### Functionality:
fixed edit page and updated packages

### Solution:
Changes done in edit-page component and in angualar.json and package.json file.

### Risk level:

### How to test:
go to my dashboard then click on task details then go to edit task you can see the task type is fixed and also task title cannot be kept empty.
